CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 * [#7317](https://github.com/rubocop-hq/rubocop/pull/7317): Add new formatter `pacman`. ([@crojasaragonez][])
 * [#6075](https://github.com/rubocop-hq/rubocop/issues/6075): Support `IgnoredPatterns` option for `Naming/MethodName` cop. ([@koic][])
 * [#7335](https://github.com/rubocop-hq/rubocop/pull/7335): Add todo as an alias to disable. `--disable-uncorrectable` will now disable cops using `rubocop:todo` instead of `rubocop:disable`. ([@desheikh][])
+* [#7396](https://github.com/rubocop-hq/rubocop/issues/7396): Display assignments, branches, and, conditions values with the offence. ([@avmnu-sng][])
```

Comment:
Move this to the `master` branch. I'd put it under `Changes` there.

Reply:
Done!